hail/python/hailtop/aiocloud/aiogoogle/client/storage_client.py

```diff
@@ -564,13 +584,8 @@ class GoogleStorageAsyncFS(AsyncFS):
 
     def __init__(self, *,
                  storage_client: Optional[GoogleStorageClient] = None,
-                 project: Optional[str] = None,
                  **kwargs):
         if not storage_client:
-            if project is not None:
```

Comment:
@daniel-goldstein you gotta fix copy.py line 118 as well.

Reply:
@danking What do you mean here? I changed copy.py line 118 to use `gcs_requester_pays_configuration`, which I thought was correct.

Reply:
I must've been looking at an old test